common/survey_utilities.py

```diff
@@ -209,30 +212,23 @@ def make_email_address(local_part=None, domain=None):
     if not domain:
         domain = local_part
 
-    return common_utilities.concatenate_strings(common_utilities.concatenate_strings(local_part, '@'),
-                                                common_utilities.concatenate_strings(domain, '.com'))
+    return f"{local_part}@{domain}.com"
```

Comment:
general comment affecting multiple changes:  prefer a single method of joining strings with an optional separator, but if this is the python way then so be it

Reply:
f-strings are a lot faster than a roll-your-own and tend to be immediately readable.